global.go

```diff
@@ -224,11 +224,15 @@ func (gm *globalManager) broadcastPeers(ctx context.Context, updates map[string]
 		SetBehavior(&rl.Behavior, Behavior_GLOBAL, false)
 		rl.Hits = 0
 
-		status, err := gm.instance.getLocalRateLimit(ctx, rl)
+		misleadingStatus, err := gm.instance.getLocalRateLimit(ctx, rl)
```

Comment:
did you confirm with the test that this returns with the correct result? I saw various places where the "fix" could be made at the time but it's hard to know if there is any knock on effects elsewhere without knowing the codebase more thoroughly.

Hopefully it is this simple :)

Reply:
I've only stared at the codebase for a couple of hours haha. As I mentioned here https://github.com/mailgun/gubernator/issues/208#issuecomment-1902264235 the other place where I thought we could fix this is in `algorithms.go`.. but I am not comfortable changing that code because it doesn't have any unit tests 😬